distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc

```diff
@@ -61,4 +61,12 @@ These are the attributes common to any of the media file types which support add
 * `contact.creator-name`
 * `contact.publisher-name`
 * `contact.point-of-contact-phone`
-* `topic.keyword`
\ No newline at end of file
+* `topic.keyword`
+
+Some file types support attributes unique to that individual type:
+
+.Additional Unique Attributes by File Type
+Mp4:: `ext.mp4.audio-sample-rate`
```

Comment:
It is an odd case. It should probably match the formatting of other file types, but it's weird that it has a list of one item.